.github/workflows/CD.yml

```diff
@@ -0,0 +1,48 @@
+
+jobs:
+  readmeWorkflow:
+    name: Unity Test Workflow ✨
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+          
+#      # Cache
+#      - uses: actions/cache@v1.1.0
+#        with:
+#          path: test-project/Library
+#          key: Library-test-project-WebGL
+
+      # Test
+      - name: Run tests
+        uses: webbertakken/unity-test-runner@v1.3
+        with:
+          projectPath: BunnyHopMaster
+          unityVersion: 2019.2.11f1
+
+      # Build
+      - name: Build project
+        uses: webbertakken/unity-builder@v0.10
+        with:
+          projectPath: BunnyHopMaster
+          unityVersion: 2019.2.11f1
+
```

Comment:
I might be wrong but maybe there are errors because our unity version for bunnyhop is 2019.3.0f3.

Reply:
I'll try that version! Thanks <3

Reply:
oof ...

https://github.com/marketplace/actions/unity-test-runner#unityversion
> unityVersion
> 
> Version of Unity to use for testing the project.
> 
> required: false default: 2019.2.1f11

Reply:
Found: https://hub.docker.com/layers/gableroux/unity3d/2019.3.9f1/images/sha256-ca190e0394accbf3bcf2cd15db08cfd1296e96f6e2888e33b31e4ba7b0f92833?context=explore

[From](https://hub.docker.com/r/gableroux/unity3d/tags)